packages/docs/src/components/content/ContentRenderer.tsx

```diff
@@ -66,6 +67,12 @@ const TextWithMaxWidth = (props: any, Component) => {
  * Each mapping has a key with the element name and a value of a functional component to be used for that element
  */
 const customComponents = (theme) => ({
+  a: (props) => {
+    if (props.href.startsWith('http')) {
```

Comment:
I think this needs to be more discriminating and not put the external link treatment on `.gov` sites like `healthcare.gov`.

Reply:
Did it in [670e9cb](https://github.com/CMSgov/design-system/pull/2522/commits/670e9cbe2880975dd1364a06d8783c14ddefad09)